libfoundation/src/system-file-posix.cpp

```diff
@@ -364,8 +364,6 @@ __MCSFileSetContents (MCStringRef p_native_path,
 
 	if (0 != rename (*t_temp_path_sys, *t_path_sys))
 	{
-		int t_save_errno = errno;
-
```

Comment:
@montegoulding @runrevmark Is correct; "indent with tabs, align with spaces" is correct no matter what your tab stop is set to. If you look at a file which has been entirely written by me with my crazy tools, then you will see that it looks correct whether it uses [2-space tabs](https://github.com/livecode/livecode/blob/develop/toolchain/lc-compile/src/lc-run.cpp?ts=2#L119) or [8-space tabs](https://github.com/livecode/livecode/blob/develop/toolchain/lc-compile/src/lc-run.cpp?ts=8#L119).